### PR TITLE
Change names for relations that lookup operand and attribute of an operation.

### PR DESCRIPTION
**Pull request by bgogul**

### CODE REVIEW

File: src/analysis/souffle/operations.dl [hunk 1]
```diff
@@ -41,29 +41,29 @@ isOperation([owner, operator, result, [input1, [input2, nil]], attrs])
...
-.decl operationToPartialOperandLists(operation: Operation, partialList: OperandList)
+.decl operationHasOperandHelper(operation: Operation, partialList: OperandList)
 
-operationToPartialOperandLists([owner, op, result, operandList, attrs], operandList) :-
+operationHasOperandHelper([owner, op, result, operandList, attrs], operandList) :-
   isOperation([owner, op, result, operandList, attrs]).
-operationToPartialOperandLists(operation, tail) :-
-  operationToPartialOperandLists(operation, [_, tail]).
+operationHasOperandHelper(operation, tail) :-
+  operationHasOperandHelper(operation, [_, tail]).
```
Comment (markww) [inline src/analysis/souffle/operations.dl:49]: Not wild about renaming this to just "helper": the partial suffixes currently reflected in the relation name allow this relation to be sensibly reused for multiple purposes without impacting readability. With the renaming, it is closely tied to this one purpose only.

We aren't using it for anything else right now, but I am going to reuse this partial state to calculate the index at which various operands reside in a future PR.

File: src/analysis/souffle/operations.dl [hunk 2]
```diff
@@ -41,29 +41,29 @@ isOperation([owner, operator, result, [input1, [input2, nil]], attrs])
...
 
 // A helper relation that produces all partial attribute lists of an
 // `Operation`.
-.decl operationToPartialAttributeLists(operation: Operation, attributes: AttributeList)
-operationToPartialAttributeLists([owner, op, result, operandList, attrs], attrs) :-
+.decl operationHasAttributeHelper(operation: Operation, attributes: AttributeList)
+operationHasAttributeHelper([owner, op, result, operandList, attrs], attrs) :-
   isOperation([owner, op, result, operandList, attrs]).
-operationToPartialAttributeLists(op, tail) :- operationToPartialAttributeLists(op, [_, tail]).
+operationHasAttributeHelper(op, tail) :- operationHasAttributeHelper(op, [_, tail]).
```
Comment (markww) [inline src/analysis/souffle/operations.dl:62]: Similar comment about "helper" here.

File: src/analysis/souffle/operations.dl (hunk 1 above)
Comment (bgogul) [reply src/analysis/souffle/operations.dl]: Changed. Made it `operandHasOperandListSuffix`. PTAL.

File: src/analysis/souffle/operations.dl (hunk 2 above)
Comment (bgogul) [reply src/analysis/souffle/operations.dl]: will resolve in the other thread.

**Comment by bgogul** [comment]
@markww PTAL.

**Pull request closed without merging**